docs/sql/statements/copy-from.rst

```diff
@@ -386,203 +420,254 @@ The ``WITH`` clause supports the following options:
 
 .. _sql-copy-from-bulk_size:
 
-``bulk_size``
-'''''''''''''
+**bulk_size**
+  | *Type:*    ``integer``
+  | *Default:* ``10000``
+  | *Optional*
 
-CrateDB will process the lines it reads from the ``path`` in bulks. This option
-specifies the size of one batch. The provided value must be greater than 0, the
-default value is 10000.
+  CrateDB will process the lines it reads from the ``path`` in bulks. This option
+  specifies the size of one batch. The provided value must be greater than 0.
 
 
 .. _sql-copy-from-fail_fast:
 
-``fail_fast``
-'''''''''''''
+**fail_fast**
+  | *Type:*    ``boolean``
+  | *Default:* ``false``
+  | *Optional*
 
-A boolean value indicating if the ``COPY FROM`` operation should abort early
-after an error. This is best effort and due to the distributed execution, it
-may continue processing some records before it aborts.
-Defaults to ``false``.
+  A boolean value indicating if the ``COPY FROM`` operation should abort early
+  after an error. This is best effort and due to the distributed execution, it
+  may continue processing some records before it aborts.
 
 .. _sql-copy-from-wait_for_completion:
 
-``wait_for_completion``
-'''''''''''''''''''''''
+**wait_for_completion**
+  | *Type:*    ``boolean``
+  | *Default:* ``true``
+  | *Optional*
 
-A boolean value indicating if the ``COPY FROM`` should wait for
-the copy operation to complete. If set to ``false`` the request
-returns at once and the copy operation runs in the background.
-Defaults to ``true``.
+  A boolean value indicating if the ``COPY FROM`` should wait for
+  the copy operation to complete. If set to ``false`` the request
+  returns at once and the copy operation runs in the background.
 
 .. _sql-copy-from-shared:
 
-``shared``
-''''''''''
+**shared**
+  | *Type:*    ``boolean``
+  | *Default:* Depends on the scheme of each URI.
+  | *Optional*
 
-This option should be set to true if the URIs location is accessible by more
-than one CrateDB node to prevent them from importing the same file.
+  This option should be set to true if the URIs location is accessible by more
+  than one CrateDB node to prevent them from importing the same file.
 
-The default value depends on the scheme of each URI.
-
-If an array of URIs is passed to ``COPY FROM`` this option will overwrite the
-default for *all* URIs.
+  If an array of URIs is passed to ``COPY FROM`` this option will overwrite the
+  default for *all* URIs.
 
 
 .. _sql-copy-from-node_filters:
 
-``node_filters``
-''''''''''''''''
+**node_filters**
+  | *Type:* ``text``
+  | *Optional*
 
-A filter :ref:`expression <gloss-expression>` to select the nodes to run the
-*read* operation.
+  A filter :ref:`expression <gloss-expression>` to select the nodes to run the
+  *read* operation.
 
-It's an object in the form of::
+  It's an object in the form of::
```

Comment:
This and similar changes are related to https://github.com/crate/crate/pull/16468#discussion_r1756673288

Ie changing rendering to show type and default, somewhat aligned with https://cratedb.com/docs/crate/reference/en/master/sql/statements/create-repository.html#sql-create-repo-s3-params